src/Methods/Runes/GetRuneHolders.ts

```diff
@@ -0,0 +1,78 @@
+import { method } from "@valkyr/api";
+import Schema, { string } from "computed-types";
+
+import { runes } from "~Database/Runes";
+
+import { schema } from "../../Libraries/Schema";
+
+export default method({
+  params: Schema({
+    runeTicker: string,
+    pagination: schema.pagination.optional(),
+    sort: string.optional(),
+  }),
+  handler: async ({ runeTicker, pagination = {}, sort }) => {
+    pagination.limit ??= 100;
+    sort ??= "asc";
+    const _sort = sort === "asc" ? -1 : 1;
+    console.log(_sort);
+
+    const etching = await runes.getEtchingByTicker(runeTicker);
+    let totalSupply = 0n;
+    if (etching?.valid) {
+      const cap = etching.terms?.cap ?? 0n;
+      const amount = etching.terms?.amount ?? 0n;
+      const premine = etching.premine ?? 0n;
+      totalSupply = BigInt(cap) * BigInt(amount) + BigInt(premine);
+    }
+
+    const filter = {
+      runeTicker,
+      spentTxid: { $exists: false },
```

Comment:
Do we need an additional DB index for this? Seems like it may be a table scan.

Reply:
could be a good idea but we will force a reindex right?